data/json/item_groups.json

```diff
@@ -823,6 +823,13 @@
       ["purse", 40],
       ["mbag", 20],
       ["fanny", 10],
+	  ["acoustic_guitar", 20],
```

Comment:
I was worried when I saw the "lack of tabbing" comment.  Borne out here.  Notepad++, probably other reputable editors such as vim or emacs, can set the Tab key to insert spaces.  Tabs as such display differently on different editors and lead to frustrated maintainers.